### ISSUE TITLE
Thumbnail youtube video

### ISSUE THREAD

**Issue by dpizzuto** [mariohahn/MHVideoPhotoGallery#109]
When I load the photogallery with correct youtube item, the thumbnails inside gallery have grey image and length of 00:00 sec. Vimeo's video haven't this issue. 
Is it a known issue ?

**Comment by mythodeia** [comment]
maybe it is because the youtube v2 is deprecated.
https://github.com/mariohahn/MHVideoPhotoGallery/issues/95

**Comment by dpizzuto** [comment]
I saw that actually there isn't a workaraound for fix this issue, right ?

**Comment by mythodeia** [comment]
as far as i know not yet

**Comment by mariohahn** [comment]
I saw one implementing V3. But i can't find it now :(

**Comment by dpizzuto** [comment]
I'll try to find out the new v3 api. I'll let you know. Please, keep this issue opened.

**Comment by dpizzuto** [comment]
I found a solution: I had to integrate new v3 api with api key and new URLs. For now, I adapted the code for my goals that are thumbnail and video duration. As soon as possibile, I'll fork your project and push the new code with pull request. Keep issue opened until I finish the operations.

**Comment by chrismeats** [comment]
:+1: This PR fixed the same issue that I was having. Hopefully something with this gets pulled into the master repo.

Although it does seem a little funny to have to edit one of the Pod files (MHGallery.m) to add your API key. Is it possible to load this value from plist value instead?

**Comment by mariohahn** [comment]
I don't like both of this ways... Maybe something have like this a just a Property:

extern void MHGalleryCustomLocalizationBlock(NSString *(^customLocalizationBlock)(NSString *stringToLocalize));

**Comment by dpizzuto** [comment]
Yes Chris, isn't the best solution so far but was, as described, a quick & dirty solution for my purpose. A good way, in my opinion, maybe to have a property to set in a SharedObject.

**Comment by mariohahn** [comment]
property would be the best (:

**Comment by chrismeats** [comment]
I went ahead and took a stab at this using MHGalleryCustomLocalizationBlock.

**Comment by okulak** [comment]
Guys, does app support all possible youtube urls for downloading thumbnails and playing video? I found an issue with urls:
"http://youtu.be/Jmb7NgNouoM"
 or
 "https://www.youtube.com/embed/DheulkmtR7Y?vq=hd720&amp;rel=0&amp;showinfo=0&amp;modestbranding=1&amp;enablejsapi=1&amp;origin=http%3A%2F%2Fwww.shyp.com"
